### PR TITLE
add URL header `last-modified` hover text on nightly download links

### PR DESCRIPTION
**Pull request by IanButterworth**
Based on https://github.com/JuliaLang/juliaup/pull/601#issuecomment-1910755082

Does what of https://github.com/JuliaCI/julia-buildkite/issues/35 is currently possible

### CODE REVIEW

**Comment by IanButterworth** [comment]
This doesn't work in the netlify preview because of cross-domain blocking
```
Access to fetch at 'https://julialangnightlies-s3.julialang.org/bin/winnt/x86/julia-latest-win32.zip' from origin 
'https://julialang.netlify.app' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the
requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource 
with CORS disabled.
```

But I believe it should from the actual website..

File: downloads/nightlies.md [hunk 1]
```diff
@@ -48,4 +51,40 @@ Most users are advised to use the latest official release version of Julia.
...
+
+<script>
+document.addEventListener("DOMContentLoaded", function () {
+  // Get the table with class "downloads"
+  var downloadsTable = document.querySelector(".downloads");
+
+  // Check if the table is found
+  if (downloadsTable) {
+    // Get all the links in the table
+    var links = downloadsTable.querySelectorAll("a");
```
Comment (DilumAluthge) [inline downloads/nightlies.md:63]: Out of curiosity, how many elements does `links` contain?

File: downloads/nightlies.md [hunk 2]
```diff
@@ -48,4 +51,40 @@ Most users are advised to use the latest official release version of Julia.
...
+
+    // Iterate through each link
+    links.forEach(function (link) {
+      // Get the href attribute of the link
+      var href = link.getAttribute("href");
+
+      if (href.startsWith("https://julialangnightlies-s3")) {
```
Comment (DilumAluthge) [inline downloads/nightlies.md:70]:
```suggestion
      if (href.startsWith("https://julialangnightlies-s3.julialang.org/")) {
```

File: downloads/nightlies.md (hunk 1 above)
Comment (IanButterworth) [reply downloads/nightlies.md:63]: 9

**Comment by IanButterworth** [comment]
I'll give it a go and revert if it doesn't work on the site

**Review by DilumAluthge — APPROVED** [review]

**Comment by inkydragon** [comment]
https://github.com/JuliaLang/www.julialang.org/actions/runs/7663122653/job/20885428964#step:12:81
```
To github.com:JuliaLang/www.julialang.org
 ! [remote rejected] github-pages-deploy-action/ucbtyhqo7 -> gh-pages (protected branch hook declined)
error: failed to push some refs to 'github.com:JuliaLang/www.julialang.org'
```

**Comment by IanButterworth** [comment]
Huh. The last 4 commits to main have failed to publish

**Comment by DilumAluthge** [comment]
From Slack: should be fixed now.

**Comment by IanButterworth** [comment]
The site deployed but CORS isn't allowing the requests
```
Access to fetch at 'https://julialangnightlies-s3.julialang.org/bin/linux/i686/julia-latest-linux-i686.tar.gz' 
from origin 'https://julialang.org' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' 
header is present on the requested resource. If an opaque response serves your needs, set the request's mode
to 'no-cors' to fetch the resource with CORS disabled.
```

Would someone be able to configure the `https://julialangnightlies-s3.julialang.org` server with something like this? 
```
Access-Control-Allow-Origin: https://julialang.org
Access-Control-Allow-Methods: GET, POST, OPTIONS
Access-Control-Allow-Headers: Content-Type, Authorization
```
At least I think it has to be that server that allows requests?

Or should we just revert?

**Comment by DilumAluthge** [comment]
@staticfloat ^

**Comment by staticfloat** [comment]
Done, seems to be working now

**Comment by IanButterworth** [comment]
Cool. Thanks!

**Comment by IanButterworth** [comment]
Weirdly it's not working for one of the links
https://julialangnightlies-s3.julialang.org/bin/linux/x86_64/julia-latest-linux-x86_64.tar.gz

```
Access to fetch at 'https://julialangnightlies-s3.julialang.org/bin/linux/x86_64/julia-latest-linux-x86_64.tar.gz' 
from origin 'https://julialang.org' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header 
is present on the requested resource. If an opaque response serves your needs, set the request's mode to 
'no-cors' to fetch the resource with CORS disabled.
```

Not a big deal, but I thought I'd report in case you know why

**Comment by staticfloat** [comment]
Probably a cached response. Try again later.

**Pull request merged**